test/e2e/config/docker.yaml

```diff
@@ -237,14 +237,12 @@ variables:
   # allowing the same e2e config file to be re-used in different Prow jobs e.g. each one with a K8s version permutation.
   # The following Kubernetes versions should be the latest versions with already published kindest/node images.
   # This avoids building node images in the default case which improves the test duration significantly.
-  KUBERNETES_VERSION_MANAGEMENT: "v1.28.0"
-  # TODO when bumping this version: also remove the pinning of the conformanceImage at the [ipv6] tests in
-  # `test/e2e/quick_start_test.go`.
-  KUBERNETES_VERSION: "v1.28.0"
-  KUBERNETES_VERSION_UPGRADE_FROM: "v1.27.3"
-  KUBERNETES_VERSION_UPGRADE_TO: "v1.28.0"
-  ETCD_VERSION_UPGRADE_TO: "3.5.9-0"
-  COREDNS_VERSION_UPGRADE_TO: "v1.10.1"
+  KUBERNETES_VERSION_MANAGEMENT: "v1.29.0"
+  KUBERNETES_VERSION: "v1.29.0"
+  KUBERNETES_VERSION_UPGRADE_FROM: "v1.28.0"
```

Comment:
Q: should be use v1.28.4 instead? (it is ok to do this as a follow up)

Reply:
There seems to be no kind image for it